readme.md

```diff
@@ -28,35 +28,19 @@ Please make sure you install the following tools before starting with the instal
 
 ## Installation
 
-> Note that you're free to adjust the `~/Sites/pastebin` location to any directory you want on your machine.
+> Note that you're free to adjust the `~/Code/pastebin` location that will be in the `Homestead.yaml` to any directory you want on your machine.
 
 1. Clone this repository: `git clone git@github.com:laravelio/pastebin.git pastebin`
-2. Add the `Homestead.yaml` file from below to the root of your project
+2. Run `./vendor/bin/homestead make`
```

Comment:
The example flag is to generate a sample `Homestead.yaml.example` file that a user can review and/or compare to the `Homestead.yaml` they're editing. By default the `make` command will use the example file as a template if it exists.

What we do need to edit here is add the no after flag so the user doesn't create the `after.sh` file (it's created by default)